Makefile

```diff
@@ -112,9 +112,10 @@ clean.nones:
 
 clean.all: clean.nones
 	@rm -rf staging/
-	# Don't mess with other registries for some semblance of a safe nuke.
+	# Don't mess with other registries for some semblance of a safe clean.
 	@docker rmi -f \
-		$(shell docker images | egrep "^$(REGISTRY)/daemon(-base)? " | awk '{print $$3}' | uniq) || true
+		$(shell docker images | egrep "^$(REGISTRY)/daemon(-base)? " | \
```

Comment:
Can you explain in what | uniq is "broken" ?
Please make the commit message explicit on that point.

Reply:
The single quotes on the `clean.nones` were misplaced. It seems the `uniq` functionality was added but not fully tested.

Reply:
We saw it, I though it was fixed.

Reply:
I see it was fixed at some point.